services/carpet-cleaning/index.html

```diff
@@ -30,7 +30,7 @@
 	<style id='elementor-frontend-inline-css' type='text/css'>
 		.elementor-74 .elementor-element.elementor-element-bee4d41:not(.elementor-motion-effects-element-type-background)>.elementor-column-wrap,
 		.elementor-74 .elementor-element.elementor-element-bee4d41>.elementor-column-wrap>.elementor-motion-effects-container>.elementor-motion-effects-layer {
-			background-image: url("../../wp-content/uploads/2020/08/logo-bg2.svg");
+			background-image: url("../../wp-content/uploads/2020/08/carpet-clean-page.jpg");
```

Comment:
Do you think we should put the full URL https://thecarpetandovenwizard.co.uk/wp-content/uploads/2020/08/carpet-clean-page.jpg ?